core/rfb.js

```diff
@@ -2208,7 +2224,14 @@ export default class RFB extends EventTargetMixin {
         this._setDesktopName(name);
         this._resize(width, height);
 
-        if (!this._viewOnly) { this._keyboard.grab(); }
+        if (!this._viewOnly) {
+            this._keyboard.grab();
+            (async () => {
+                if (await isAsyncClipboardAvailable()) {
+                    this._clipboard.grab();
+                }
+            })();
```

Comment:
I think this is a bit messy. I would prefer a `.then()` approach like other places where we join sync and async code.